.github/workflows/testing-integration.yaml

```diff
@@ -0,0 +1,91 @@
+name: "Integration Tests"
+'on':
+  workflow_call: {}
+
+jobs:
+  data-plane-serverless:
+    name: Data plane serverless integration tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        use_grpc: [true, false]
+        metric: 
+          - cosine
+          # - euclidean
+          # - dotproduct
```

Comment:
Assuming you want to keep these commented out for now, but if not.

Reply:
Yeah, leaving them out while still fleshing out the tests and iterating heavily. Dotproduct is having some kind of performance issue at the moment the db team are looking into.